ui/dasher/src/sound.ts

```diff
@@ -63,19 +69,76 @@ export class SoundCtrl extends PaneCtrl {
               h(
                 'button.text',
                 {
-                  hook: bind('click', () => this.set(s[0])),
+                  hook: bind('click', () => {
+                    if (s[0] === 'speech') {
+                      this.showVoiceSelection = true;
+                      if (current === 'speech') this.redraw();
+                    }
+                    if (s[0] !== current) this.set(s[0]);
+                  }),
                   class: { active: current === s[0] },
                   attrs: { 'data-icon': licon.Checkmark, type: 'button' },
                 },
-                s[1],
+                [s[1], s[0] === 'speech' ? '...' : ''],
               ),
             ),
           ),
         ]),
+        this.showVoiceSelection
+          ? snabDialog({
+              attrs: {
+                dialog: {
+                  role: 'dialog',
+                },
+              },
+              class: '',
+              css: [],
```

Comment:
```suggestion
```

Reply:
the 'dialog' role on a dialog element seems redundant, but i'm open to evidence to the contrary